### DISCUSSION TITLE
Access Sunsynk by Web Service API

### DISCUSSION THREAD

**Discussion by Mike-CPT** [kellerza/sunsynk#237] [Q&A]
Hi there,
I checked and installed this adapter. But I am not sure, if I am following the right route.
Is this adapter using a local integration into the Sunsynk inverter? 
Since my inverter is located in a different network and I don't have access to that network, I am looking for a solution that uses the Sunsynk API in the cloud (the same API the sunsynk app uses). There is a Sunsynk dev guide available. Is this adapter based on that API and would this adapter allow me to access the data/information for my inverter from the Sunsynk?
Mike.

**Comment by kellerza** [comment]
This plugin only uses local access to the Sunsynk dongle/via an RS485 interface
